js/views/components/moderators/Status.js

```diff
@@ -38,7 +38,7 @@ export default class extends BaseVw {
         let mode = this.getState().mode;
         if (mode === 'loadingXofY') mode = 'loadingXofYTimedOut';
         this.setState({ showSpinner: false, mode });
-      }, 10000);
+      }, 60000);
```

Comment:
I made this longer since I was seeing the message for no mods appear then mods would load, which was pretty clunky.